### PR TITLE
Task/jbook clone

### PR DESCRIPTION
**Pull request by JLyons1985**
## !vibez

This one is massive...... Jbook clone massive. Sorry. This will be up on 72 for testing.

## Related Issue/Ticket

Pretty much all the jbook tickets

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Smoke testing instructions

Basically test everything. Still working on writing tests for the backend stuff so this is in the works.

## Checklist:

- [ ] Documentation updated
- [ ] Unit tests added/updated
- [ ] Smoke testing relevant to authentication needed
- [ ] Clones involved and accounted for
- [ ] RDS migrations or other data manipulation necessary
- [ ] Dev tools or other infrastructure changed
- [ ] Fix User Data and favoriting not working for GC

### CODE REVIEW

**Review by ryanryanchan — DISMISSED** [review]

**Review by aaron42051 — APPROVED** [review]
a beautiful beast

**Review by ryanryanchan — APPROVED** [review]
amazing
